market_maker/ws/ws_thread.py

```diff
@@ -318,11 +324,11 @@ def findItemByKeys(keys, table, matchData):
         if matched:
             return item
 
+
 if __name__ == "__main__":
     # create console handler and set level to debug
     logger = log.setup_custom_logger('websocket', logging.DEBUG)
     ws = BitMEXWebsocket()
     ws.connect("https://testnet.bitmex.com/api/v1")
-    while(ws.ws.sock.connected):
+    while (ws.ws.sock.connected):
```

Comment:
Nit: Please can we remove unrelated formatting changes (if you'd like to make them please do so in a separate PR)

Reply:
Thanks. Addressed in be0687c6833157dbe5441472e77c0e2532e4f419